### PR TITLE
Integrate gpu simulator as service

### PR DESCRIPTION
**Pull request by paaige**

### CODE REVIEW

File: cirq_superstaq/service.py [hunk 1]
```diff
@@ -263,6 +264,22 @@ def get_backends(self) -> dict:
...
+    ) -> str:
+        serialized_circuit = css.serialization.serialize_circuits(circuit)
+
+        request_json =  {
+            "cirq_circuits": serialized_circuit,
+            "backend": target,
+            "gpu": gpu,
+        }
+
+        amplitudes = self._client.simulate(request_json)
```
Comment (vtomole) [inline cirq_superstaq/service.py:278]: Open a draft PR in gss for this method.

File: cirq_superstaq/service.py [hunk 2]
```diff
@@ -263,6 +264,22 @@ def get_backends(self) -> dict:
         """Get list of available backends."""
         return self._client.get_backends()["superstaq_backends"]
 
+    def simulate(
```
Comment (vtomole) [inline cirq_superstaq/service.py:267]: We've decided to get rid of this function in place of passing in `method` and `options` arguments to `create_job`

**Review by vtomole — CHANGES_REQUESTED** [review]

File: cirq_superstaq/service.py (hunk 2 above)
Comment (paaige) [reply cirq_superstaq/service.py:267]: that's from a very old commit

File: cirq_superstaq/service.py [hunk 3]
```diff
@@ -229,7 +233,9 @@ def create_job(
             serialized_circuits={"cirq_circuits": serialized_circuits},
             repetitions=repetitions,
             target=target,
+            method=method,
             ibmq_pulse=ibmq_pulse,
+            options={"options": json.dumps(options)},
```
Comment (vtomole) [inline cirq_superstaq/service.py:238]: Link

**Review by vtomole — COMMENTED** [review]
Need to pass checks.

**Review by vtomole — CHANGES_REQUESTED** [review]
Add tests for this `method` and `serialized_options` arguments.

**Comment by vtomole** [comment]
After that this should be good to go.

**Review by vtomole — APPROVED** [review]
LGTM

**Review by vtomole — APPROVED** [review]
LGTM!

**Pull request merged**